cloud/linode/loadbalancers.go

```diff
@@ -450,7 +450,15 @@ func (l *loadbalancers) getNodeBalancerByHostname(ctx context.Context, service *
 }
 
 func (l *loadbalancers) getNodeBalancerByIPv4(ctx context.Context, service *v1.Service, ipv4 string) (*linodego.NodeBalancer, error) {
-	lbs, err := l.client.ListNodeBalancers(ctx, nil)
+	// TODO: use this constructor once we merge in linodego v1 update
```

Comment:
Maybe we could include the linodego update in this PR and go ahead and use the new constructor.

Reply:
See my comment in the thread above (or in JIRA) as to why we (meaning me and Tyler when discussing it privately) decided to split it.